nobrainer/processing/base.py

```diff
@@ -54,7 +54,7 @@ def save(self, save_dir):
             pk.dump(model_info, fp)
 
     @classmethod
-    def load(cls, model_dir, multi_gpu=False, custom_objects=None, compile=False):
+    def load(cls, model_dir, *args, **kwargs):
```

Comment:
this is generally considered bad style to subsume everything into args and kwargs.

Reply:
Ok, I didn't realize that but I suppose it makes sense to be more explicit. It's just a lot of args to pass around. But fixed.